### PR TITLE
Mass delete contacts search bar in nav bar

### PR DESCRIPTION
**Pull request by ssoonmi**
On desktop: 
Screen Shot 2019-09-15 at 1 25 27 AM

In mobile:
Screen Shot 2019-09-15 at 1 25 35 AM

### CODE REVIEW

**Comment by ssoonmi** [comment]
search bar should span whole nav bar if the search text is long

**Comment by ssoonmi** [comment]
search input covers logo if too long

Screen Shot 2019-09-16 at 1 53 21 PM

File: layouts/default.vue [hunk 1]
```diff
@@ -91,12 +112,26 @@
...
+        get () {
+          return this.$store.state.searchInput
+        },
+        set (value) {
+          this.updateSearchInput(value)
+        }
+      },
+      logoIsShowing () {
+        if (this.$route.path !== '/listguests') return true
+        else if (!this.$vuetify.breakpoint.xs) return true
```
Comment (anjosh) [inline layouts/default.vue]: can you change to `!this.$vuetify.breakpoint.xs` to [`this.$vuetify.breakpoint.smAndUp`](https://vuetifyjs.com/en/customization/breakpoints)

File: layouts/default.vue [hunk 2]
```diff
@@ -91,12 +112,26 @@
...
+      },
+      search: {
+        get () {
+          return this.$store.state.searchInput
+        },
+        set (value) {
+          this.updateSearchInput(value)
+        }
+      },
+      logoIsShowing () {
```
Comment (anjosh) [inline layouts/default.vue]: rename to `showLogo`?

File: layouts/default.vue [hunk 3]
```diff
@@ -26,8 +26,29 @@
       </v-list>
     </v-navigation-drawer>
     <v-toolbar fixed app :clipped-left="clipped">
-      <v-toolbar-side-icon @click="toggleDrawer"></v-toolbar-side-icon>
-      <v-img src="/icon.png" height="70%" contain position="left center" class="ml-2 mt-2"/>
+      <v-layout style="height: 100%; width: 100%; justify-content: space-between;" row align-center>
```
Comment (anjosh) [inline layouts/default.vue]: Any reason to have the css in the template? How about putting it in `<style>`?

Comment (ssoonmi) [reply layouts/default.vue]: put css in <style>, let me know what you think

**Comment by anjosh** [comment]
Looks great. Thank you!

**Pull request merged**